bin/ch-convert

```diff
@@ -270,6 +300,109 @@ cv_tar_squash () {
     rm -Rf --one-file-system "$tar_dir"
 }
 
+## Dockman functions ##
+
+# Use for conversions involving docker or podman. Similarities between the
+# command line instructions for docker and podman allow to write generalized
+# functions that can be used for both (hence 'dockman'). When calling a
+# 'dockman' function, image format is specified by the first argument
+# ('docker_' or 'podman_').
+
+chimage_to_dockman () {
+    chimage_tar=${tmpdir}/weirdal.tar.gz
+    cv_chimage_tar "$2" "$chimage_tar"  # FIXME: needlessly compresses?
+    tar_to_dockman "$1" "$chimage_tar" "$3"
+    rm "$chimage_tar"
+}
+
+dir_to_dockman () {
+    dirtar=${tmpdir}/weirdal.tar.gz
+    # One could also use "docker build" with "FROM scratch" and "COPY",
+    # apparently saving a tar step. However, this will in fact tar the source
+    # directory anyway to send it to the Docker daemon.
+    cv_dir_tar "$2" "$dirtar"  # FIXME: needlessly compresses
+    tar_to_dockman "$1" "$dirtar" "$3"
+    rm "$dirtar"
+}
+
+dm_fmt_name () {
+    # Format 'docker_' and 'podman_' as 'Docker' and 'Podman' respectively.
+    case $1 in
+	docker_)
+	    echo "$1" | tr "d" "D" | tr -d "_"
+	    ;;
+	podman_)
+	    echo "$1" | tr "p" "P" | tr -d "_"
+	    ;;
+	*)
+	    FATAL "unreachable code reached"
+	    ;;
+    esac
```

Comment:
`tr dp DP` doesn't work for podman because it has both a "p" and a "d", e.g.
```
$ echo podman | tr dp DP
PoDman
```
`case` was the best alternative I could think of